apps/container/src/views/DonateView.vue

```diff
@@ -1,5 +1,265 @@
+<script setup lang="ts">
+import { ref } from 'vue';
+
+const dialog = ref(false);
+
+const tableData = ref([
+  {
+    name: 'Domínio ufabcnext.com',
+    description:
+      'É o nome da nossa URL. Os domínios servem para identificar, de forma amigável, os endereços das aplicações pela internet.',
+    amount: 'R$ 70,00/ano',
+  },
+  {
+    name: 'Servidor (Digital Ocean)',
+    description:
+      'Para a plataforma poder estar no ar, precisamos hospedar toda a interface e sua infraestrutura, assim como a sua API. Para isso, usamos um servidor na cloud DigitalOcean, o que também nos gera custos',
+    amount: 'US$ 120,00/ano',
+  },
+  {
+    name: 'Conta Apple Developer',
+    description:
+      'Precisamos pagar uma taxa anual para podermos ter uma conta de desenvolvedor na Apple, e assim poder publicar aplicativos para a Apple Store.',
+    amount: 'US$ 100,00/ano',
+  },
+]);
+</script>
+
 <template>
-  <div class="donate">
-    <h1>Donate</h1>
-  </div>
+  <v-container>
+    <v-dialog v-model="dialog" width="700px" transition="scroll-y-transition">
+      <template v-slot:activator="{ isActive, props }">
+        <v-row class="justify-center">
+          <v-col sm="12" md="12" lg="12" class="justify-center">
+            <div class="donation-text">
+              <h2 class="donate-title text-primary">Ajude o UFABC Next</h2>
+              <div class="history-text mt-3 mb-4">
+                Que bom que temos você como nosso usuário 🥰🥳 <br /><br />
+                O Next foi desenvolvido em 2019 <b>de alunos para alunos</b> e
```

Comment:
entendo que, por uma questão de semantica de HTML, o `strong` é melhor que o `b` nesse caso
```suggestion
                O Next foi desenvolvido em 2019 <b>de alunos para alunos</b> e
```